aws_lambda_builders/workflows/nodejs_npm/workflow.py

```diff
@@ -60,3 +60,9 @@ def __init__(self,
             CopySourceAction(tar_package_dir, artifacts_dir, excludes=self.EXCLUDED_FILES),
             npm_install,
         ]
+
+    def get_resolvers(self):
+        """
+        specialized path resolver that just returns the list of executable for the runtime on the path.
+        """
+        return [PathResolver(runtime=self.runtime, binary="node")]
```

Comment:
Thought node needed `npm`?

Reply:
by default, we look for the language in the workflow.. the workflow's language is filled as nodejs, but the executable is called node.